packages/app-editors/kakoune/kakoune.exlib

```diff
@@ -0,0 +1,63 @@
+# Copyright 2014 Maxime Coste <frrrwww@gmail.com>
+# Distributed under the terms of the GNU General Public License v2
+
+require toolchain-funcs github [ user=mawww tag=v${PV} ]
+
+SUMMARY="Mawww's experiment for a better code editor"
+
+LICENCES="public-domain"
+SLOT="0"
+MYOPTIONS="debug"
+
+DEPENDENCIES="
+    build+run:
+        sys-libs/ncurses[>=5.3]
+    suggestion:
+        app-text/editorconfig [[
+            description = [ ${PN} comes with an editorconfig support script ]
+        ]]
+        dev-util/ctags [[
+            description = [ ${PN} comes with a ctags support script ]
+        ]]
+"
+
+WORK=${WORK}/src
+
+pkg_pretend() {
+    if cxx-is-gcc && ! ever at_least 5 $(gcc-version); then
```

Comment:
Kakoune now requires gcc 7, could you fix that as well ?

Reply:
Fixed.